src/main/untangled/ui/elements.cljc

```diff
@@ -195,3 +195,24 @@
                        (dissoc :active :title :type)
                        clj->js)]
     (apply dom/div attributes (when title (build-title title)) children)))
+
+(defn ui-checkbox
+  "Render a checkbox (not the label). Props is a normal clj(s) map with React/HTML attributes plus:
+
+  className (optional): additional class stylings to apply to the top level of the checkbox
+  id: Name of the checkbox
+  style (optional):  is-indeterminate :c-checkbox--informative"
+  [{:keys [id style] :or {id ""} :as attrs}]
+  (let [legal-styles #{:is-indeterminate :c-checkbox--informative}
```

Comment:
Informative has been removed as it's an internal-only class we don't need in this project.